integreat_cms/cms/models/contact/contact.py

```diff
@@ -61,7 +63,19 @@ def __str__(self) -> str:
 
         :return: A readable string representation of the contact
         """
-        return f"{self.title} {self.name}"
+        point_of_contact_for = (
+            f"{self.point_of_contact_for.strip()}: "
+            if self.point_of_contact_for and self.point_of_contact_for.strip()
+            else ""
+        )
+        name = f"{self.name.strip()} " if self.name and self.name.strip() else ""
+        details = [
+            detail
+            for x in ["email", "phone_number", "website"]
+            if (detail := self.get(x)) and detail.strip()
+        ]
+        details_str = f"({', '.join(details)})" if details else ""
+        return f"{point_of_contact_for}{name}{details_str}"
```

Comment:
Wouldn't it be better to make sure in the form that the fields are already stripped, instead of having to strip them every time they are used?

Reply:
yes, you're absolutely right

Reply:
Is there still something that should be done for this? :)

Reply:
The form still needs to strip the fields. I suspect that this may be quite a common issue that also affects other models, so you could also open an issue, instead of implementing that in this pr I guess

Reply:
I think this needs some internal discussion about how we want to go about it. There are multiple ways to go about this and I think this needs to be decided in the entire team. Therefore I will open up a new issue for this